assert_release_quality

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
+#
+# Asserts that the module is ready for release with the expected quality
 
 # cf. https://docs.npmjs.com/cli/version
 new_version=$1
 
+# Checking that the npm-shrinkwrap.json file is up-to-date wrt to the
+# package.json file.
+if [ package.json -nt npm-shrinkwrap.json ]
+then
+    echo "File package.json is newer than file npm-shrinkwrap.json"
```

Comment:
@pdehaan: yes sure there are plenty of reasons to modify package.json that wouldn't require you to regenerate the npm-shrinkwrap.json file. But apart from comparing the modification time, what could we do? Also we could document that doing a manual `touch npm-shrinkwrap.json` could be done if the `package.json` was modified in other areas than the dependencies. What do you think?